erpnext/payroll/doctype/employee_tax_exemption_declaration/employee_tax_exemption_declaration.json

```diff
@@ -102,12 +104,22 @@
    "fieldname": "total_exemption_amount",
    "fieldtype": "Currency",
    "label": "Total Exemption Amount",
+   "options": "currency",
    "read_only": 1
+  },
+  {
+   "default": "Company:company:default_currency",
+   "fieldname": "currency",
```

Comment:
currency field not required here, as declaration amounts are always in company currency.